storage/mysql/log_storage.go

```diff
@@ -13,7 +13,8 @@ import (
 	"github.com/google/trillian/storage"
 )
 
-const getLogPropertiesSql string = "SELECT AllowsDuplicateLeaves FROM Trees WHERE TreeId=?"
+const getTreePropertiesSql string = "SELECT AllowsDuplicateLeaves FROM Trees WHERE TreeId=?"
```

Comment:
These three lines should probably be `...SQL` but I think there are a bunch of other consts like that, so maybe can fix those all together in a separate PR?